log/logfmt_logger.go

```diff
@@ -0,0 +1,29 @@
+package log
+
+import (
+	"io"
+
+	"gopkg.in/logfmt.v0"
+)
+
+type logfmtLogger struct {
+	w io.Writer
+}
+
+// NewLogfmtLogger returns a basic logger that encodes keyvals as simple "k=v"
+// pairs to the Writer.
+func NewLogfmtLogger(w io.Writer) Logger {
+	return &logfmtLogger{w}
+}
+
```

Comment:
Nice find -- Wasn't aware of that either. Seems unfortunate it won't appear in GoDoc as it's the most important function in this package!